BedrockCommand.cpp

```diff
@@ -206,6 +211,7 @@ void BedrockCommand::finalizeTimingInfo() {
     }
 
     SINFO("command '" << methodName << "' timing info (ms): "
+          "prePeek: " << prePeekTotal/1000 << " (count: " << prePeekCount << "), "
```

Comment:
Several places - but we haven't added postProcess.

Reply:
Added the postProcess timing.